cmd/gunicorn.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "Starting Model Server"
+export GUNICORN_PORT=$PORT_MODEL_SERVER
+gunicorn --config config/gunicorn_config.py src.server.model_server:app &
+echo "Starting Offline Trainer"
+export GUNICORN_PORT=$PORT_OFFLINE_TRAINER
+gunicorn --config config/gunicorn_config.py src.train.offline_trainer:app &
+wait
```

Comment:
Can we create separate containers for model-server and trainer? 
This removes the issue of handling (unexpected) termination.